Podcast/CardTableViewCell.swift

```diff
@@ -148,11 +151,8 @@ class CardTableViewCell: UITableViewCell {
         episodeNameLabel = UILabel(frame: CGRect.zero)
         dateTimeLabel = UILabel(frame: CGRect.zero)
         descriptionLabel = UILabel(frame: CGRect.zero)
-        recommendedLabel = UILabel(frame: CGRect.zero)
-        playLabel = UILabel(frame: CGRect.zero)
-        
         
-        let labels = [episodeNameLabel, dateTimeLabel, descriptionLabel, recommendedLabel, playLabel]
+        let labels = [episodeNameLabel, dateTimeLabel, descriptionLabel]
         for label in labels {
             label!.textAlignment = .left
```

Comment:
you should not need to force unwrap this, labels should be of type [UILabel] not [UILabel?]